SPECS/kernel/config

```diff
@@ -169,14 +174,16 @@ CONFIG_HAVE_UNSTABLE_SCHED_CLOCK=y
 #
 # Scheduler features
 #
-# CONFIG_UCLAMP_TASK is not set
+CONFIG_UCLAMP_TASK=y
+CONFIG_UCLAMP_BUCKETS_COUNT=5
```

Comment:
Uclamp monitors utilization for cpus and caps the usage
https://cateee.net/lkddb/web-lkddb/UCLAMP_TASK.html
https://cateee.net/lkddb/web-lkddb/UCLAMP_BUCKETS_COUNT.html